cincinnati/src/plugins/internal/graph_builder/dkrv2_openshift_secondary_metadata_scraper/plugin.rs

```diff
@@ -15,11 +15,11 @@ use self::cincinnati::plugins::prelude_plugin_impl::*;
 use tokio::sync::Mutex as FuturesMutex;
 
 pub static DEFAULT_OUTPUT_ALLOWLIST: &[&str] = &[
-    "/LICENSE$",
-    "/channels/.+\\.ya+ml$",
-    "/blocked-edges/.+\\.ya+ml$",
-    "/raw/metadata.json$",
-    "/version$",
+    "LICENSE$",
+    "channels/.+\\.ya+ml$",
+    "blocked-edges/.+\\.ya+ml$",
+    "raw/metadata.json$",
+    "version$",
```

Comment:
This fixes things; however, this would mean the plugin was not working for quite some time... Maybe there is somewhere a configuration that was recently misconfigured and the `DEFAULT_OUTPUT_ALLOWLIST` is correct?

Reply:
Huh, originally the initial forward slash was missing [1]. They were added later on.

The internal-channel directory was to be removed by using the slash [2].

And more files in [3].

I need to have a look at this again.

[1] https://github.com/openshift/cincinnati/commit/53b921219b10612532539b1fa72062551e1700d2
[2] https://github.com/openshift/cincinnati/commit/97dc7615f185d74c2c670ff8681a4ba18973322e
[3] https://github.com/openshift/cincinnati/commit/9d6e9d9146b16103a997003a9aa018e8dd8d455d

Reply:
Resolved.